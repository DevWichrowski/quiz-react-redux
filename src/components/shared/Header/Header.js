import React from 'react';
import "./Header.scss";
import PropTypes from "prop-types";

const Header = ({headerText, style}) => {
    return (
        <div className="header" style={style}>
            <h2>{headerText}</h2>
        </div>
    );
};

Header.propTypes = {
    headerText: PropTypes.string.isRequired,
    style: PropTypes.object
};

export default Header;