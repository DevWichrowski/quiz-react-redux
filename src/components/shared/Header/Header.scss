@import "./src/styles/colors";

.header {
  background-color: $transparent-gray;
  border-top-right-radius: 10px;
  border-top-left-radius: 10px;
  padding: 15px;
  text-align: center;

  h2 {
    margin: 0;
  }
}