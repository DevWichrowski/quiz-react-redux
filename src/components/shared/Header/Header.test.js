import {testComponentRender} from "../../../core/helpers/tests/test-component-render";
import React from "react";
import Header from "./Header";

describe('Header', () => {
    testComponentRender(<Header/>)
});
