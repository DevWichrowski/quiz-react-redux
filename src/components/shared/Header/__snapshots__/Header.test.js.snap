// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
<div
  className="header"
>
  <h2>
    React Quiz
  </h2>
</div>
`;
